src/app/api/auth.api.ts

```diff
@@ -48,6 +52,21 @@ export class AuthApi {
         this.userChanges$.next(user);
     }
 
+    public fetchDatasetsByAccountName(
```

Comment:
This query is not about authentication, it's about the datasets, so please move it to the datasets API part.